packages/Responsive/Responsive.jsx

```diff
@@ -4,12 +4,7 @@ import Media from 'react-media'
 
 import { warn } from '../../shared/utils/warn'
 
-const breakpoints = {
-  sm: 576,
-  md: 768,
-  lg: 992,
-  xl: 1200,
-}
+import breakpoints from './breakpoints'
```

Comment:
As a suggestion, you can always export small amounts of code without needing to create new files. 

```diff
- const breakpoints = {
+ export const breakpoints = {
  sm: 576,
  md: 768,
  lg: 992,
  xl: 1200,
}
```